test/exporter.js

```diff
@@ -0,0 +1,80 @@
+import test from 'ava';
+import sinon from 'sinon';
+
+import 'babel-register';
+import 'babel-polyfill';
+
+import Exporter from '../src/exporter';
+import { checksum } from '../src/helpers';
+
+let dbMock;
+let zipMock;
+let exporter;
+
+test.beforeEach(() => {
+  dbMock = {
+    export: () => "data",
+    prepare: () => ({getAsObject: () => null })
+  };
+  zipMock = {
+    file: () => null,
+    generateAsync: () => null
+  };
+  exporter = new Exporter(dbMock, zipMock);
+});
+
+test('Exporter class exists', t => {
```

Comment:
Same as https://github.com/ewnd9/anki-apkg-export/pull/8/files#r90220231 

`new Exporter !== new Exporter` It's JavaScript after all, so everything is possible. But, hypothetically, what is the point of overriding `new` behavior?